en/contribute/jobs/index.md

```diff
@@ -36,9 +52,9 @@ the following, and want to be part of our development team.
 * Python for server development
 * Docker and dev/ops
 
-### Let's define the role together
-You can be a junior developer fresh out of school, a capable hobbyist or an
-experienced senior. It's not important whether your primary skills are front-end
+## Let's define the role together
+You can be a junior fresh out of school, a capable hobbyist or an experienced
+senior. It's not important whether your primary skills are design, front-end
 or back-end development, mobile or web.
 
 For us the most important thing is to find someone who shares our progressive
```

Comment:
I like this! I just really don't want to scare (or put) anyone off because our target group is so niche, so anyone we can get in contact with is good, even if they only have one or two projects online since before.

Will try to work some of this in though! Or could maybe use it as a first response to those who reach out.